apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styles from './chatMessage.module.scss';
+import classNames from 'classnames';
+
+interface ChatMessageProps {
+  id: string;
+  name: string;
+  role: string;
+  chatMessageText: string;
+  status: string;
+}
+
+const INAPPROPRIATE_MESSAGE = 'This message has been flagged as inappropriate.';
+const TOO_PERSONAL_MESSAGE = 'This message has been flagged as too personal.';
```

Comment:
Localization for these as well? ok if this is being handled as a follow up

Reply:
I'll go ahead and add in this PR.